### DISCUSSION TITLE
Index only new files/folders

### DISCUSSION THREAD

**Discussion by xhabih** [photoprism/photoprism#1702] [Ideas]
I miss an option to index only new items.

It is also very cumbersome for me to manually select for indexing a new created folder since I have to scroll all the way down (and I have a lot of folders!).

**Comment by lastzero** [comment]
When you go to Library > Index and click Start, only new items will be indexed by default. Everything else is skipped unless you've selected "Complete Rescan".

**Reply by xhabih** [reply]
OK, sorry, I'm stupid.

**Reply by lastzero** [reply]
No worries :)
